plugins/orchestrator/src/components/WorkflowInstancePage.tsx

```diff
@@ -204,7 +209,7 @@ export const WorkflowInstancePage = ({
                     <Button
                       variant="contained"
                       color="primary"
-                      disabled={!canAbort}
+                      disabled={!permittedToExecute.allowed || !canAbort}
```

Comment:
we should probably use the orchestratorWorkflowInstanceAbortPermission and same for line 222